openedx/core/djangoapps/content/course_overviews/migrations/0008_drop_course_tabs_fk_constraint.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Foreign keys aren't enforced by default on our version of SQLite3, and
+        # trying to delete them will throw an error. See:
+        #   http://south.aeracode.org/ticket/775
+        if db.backend_name != 'sqlite3':
+            db.delete_foreign_key('course_overviews_courseoverviewtab', 'course_overview_id')
```

Comment:
So how does this work when the `CourseOverviewTab` model still lists the foreign key relationship?
https://github.com/edx/edx-platform/pull/10269/files#diff-b01875180f0e688c01b7ff8c55ec56dbR59

Also, will the related tab entries still be deleted when its corresponding course overview is deleted?  This was the case before.